src/templates/zh/2019/accessibility_statement.html

```diff
@@ -0,0 +1,122 @@
+{% extends "base/2019/accessibility_statement.html" %}
+
+{% block title %}无障碍访问声明 | The Web Almanac 源于 HTTP Archive{% endblock %}
+
+{% block description %}描述Web Almanac的无障碍访问目标{% endblock %}
+
+{% block twitter_image_alt %}Web Almanac 无障碍访问声明{% endblock %}
+
+{% block date_published %}2020-02-03T12:00:00.000Z{% endblock %}
+{% block date_modified %}2020-06-21T00:00:00.000Z{% endblock %}
+
+{% block index %}
+      <ul>
+        <li><a href="#overview">概览</a></li>
+        <li><a href="#measures">支援无障碍访问的措施</a></li>
+        <li><a href="#conformance-status">一致性状态</a></li>
+        <li><a href="#external-content">外部内容</a></li>
+        <li><a href="#compatibility">与浏览器和辅助技术的兼容性</a></li>
+        <li><a href="#technical-specifications">技术规格</a></li>
+        <li><a href="#assessment-approach">评估方法</a></li>
+        <li><a href="#feedback">反馈</a></li>
+      </ul>
+{% endblock %}
+
+{% block main_content %}
+      <section>
+        <h2 id="overview"><a href="#overview" class="anchor-link">概览</a></h2>
+
+        <p>
+          Web Almanac网络年鉴致力于确保对所有人的数字无障碍化。我们在不断改进每个人的用户体验，并应用相关的无障碍访问标准。
```

Comment:
Maybe you mean traditional Chinese ? Because I seldom heard 数位 in life， Baidu search https://www.baidu.com/s?wd=%E6%95%B0%E5%AD%97%E6%97%A0%E9%9A%9C%E7%A2%8D%E5%8C%96&rsv_spt=1&rsv_iqid=0xdbbee8710010970d&issp=1&f=8&rsv_bp=1&rsv_idx=2&ie=utf-8&rqlang=cn&tn=baiduhome_pg&rsv_enter=1&rsv_dl=tb&oq=%25E6%2595%25B0%25E4%25BD%258D%25E6%2597%25A0%25E9%259A%259C%25E7%25A2%258D%25E5%258C%2596&rsv_btype=t&inputT=1508&rsv_t=3f9bOq2cE%2F2oCSSJt%2FOO%2FJjM7xvvEimwyrhUXDMgjGcBGtaJeRYHegR7daF1Ep38yKLa&rsv_pq=dfb6a3b90011d199&rsv_sug3=30&rsv_sug2=0&rsv_sug4=32849                  Sohu news: https://www.sohu.com/a/279732253_727515

Reply:
Going to quote from what I wrote in the [Translator’s Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide) here:

> There will be differences and disagreements between translations. Languages are fluid and spoken in many different places and ways around the world. We will leave it to the translation teams to try to reach consensus. Where either phrasing could be used, the pull request author has final say (reviews are suggestions not dictates) but would strongly advise trying to get consensus.

Reply:
Thank you @bazzadp and  @AbbyTsai !  I think there's a problem ，There are two Chinese character standard ：Simplified(China mainland,Singapore,Malaysia)    tranditional (Hongkong,Macau,Taiwan) 
  https://en.wikipedia.org/wiki/Simplified_Chinese_characters  (eg:  https://www.bbc.com/zhongwen/simp    https://www.bbc.com/zhongwen/trad )  ,  So the Characters and many foreign words are very different. I'll discuss with @AbbyTsai , there may two solutions , translate to 1 standard first, than use some tools to make it to another standard,  second, may create a new language folder (zh-cn zh-hk) from begining to translate to 2  standards?   maybe option 1 could save time.

Reply:
OK, will let you two decide what’s best, but may want to rename the folder to whichever you decide to make it clear what this is, rather than just `zn`. Not sure what the convention normally is for other sites that have the language in the URL like we do.